frontend/src/components/explorer/mapViewer/SidebarDataPanels.vue

```diff
@@ -7,28 +7,45 @@
           <i>{{ currentMap.name }}</i>
         </p>
       </header>
-      <div v-if="currentMap.reactionList && missingReactionList.length > 0"
+      <div v-if="dim==='2d' && currentMap.reactionList && missingReactionList.length > 0"
            class="card-content p-4">
-        <div class="content mb-0">
+        <div v-if="currentMap.mapReactionIdSet.length == 1" class="content mb-0">
           Please note that {{ missingReactionList.length }}
           of the reactions in the {{ currentMap.type }} are not shown on the map.
           <a @click="showModal = true"> See comparison </a>
         </div>
+        <div v-else class="content mb-0">
+          Please note that {{ missingReactionList.length }} of the reactions in the
+          {{ currentMap.type }} are not shown on any of the {{ currentMap.name }} maps.
+          <a @click="showModal = true"> See comparison </a>
+        </div>
         <div v-if="showModal" class="modal is-active">
           <div class="modal-background" @click="closeModal"></div>
           <div class="modal-content p-5 column is-6-fullhd is-8-desktop is-10-tablet is-full-mobile
           has-background-white">
             <h4 class="title is-size-4 m-0 mb-2"> List of missing and total reactions on the map </h4>
-            <p class="pb-4">
+            <p v-if="currentMap.mapReactionIdSet.length == 1" class="pb-4">
               There are {{ missingReactionList.length }} reactions not shown on the map. Some reactions
               are missing as the {{ currentMap.type }} is being updated much more often than the maps.
               Also, as the maps are manually curated, occasionally some reactions cannot be added.
-              The number of reactions shown are {{ mapReactionList.length }}.
+              The number of reactions shown is {{ mapReactionList.length }}.
+            </p>
+            <p v-else class="pb-4">
+              There are {{ missingReactionList.length }} reactions not shown on any of the {{ currentMap.name }} maps.
+              The 2D map of {{ currentMap.name }} is split into multiple maps due to its size. The number of missing
+              reactions displayed is the missing reactions when the reactions in all {{ currentMap.name }} maps are
+              combined.
+              Some reactions are missing as the {{ currentMap.type }} is being updated much more often than the maps.
+              Also, as the maps are manually curated, occasionally some reactions cannot be added.
+              The number of reactions shown is {{ mapReactionList.length }}.
```

Comment:
@mihai-sysbio do you think we should add information regarding the majority of missing reactions being transport reactions? Or will that just clutter the information?

Reply:
Great idea!

Reply:
I don't think it would clutter the info, but yeah it might be faster to merge this in without additional changes.

Reply:
I don't think it would clutter the info, but yeah it might be faster to merge this in without additional changes.

Reply:
Created issue #650 to cover this